### PR TITLE
Dutch translations

### PR DESCRIPTION
**Pull request by Alagoss**

### CODE REVIEW

File: modDesc.xml [hunk 1]
```diff
@@ -91,5 +103,11 @@
             <fr><![CDATA[Allemagne]]></fr>
```
Comment (bbenoist) [inline modDesc.xml:104]: @Alagoss How do you say Germany and France in Dutch?

**Review by bbenoist — CHANGES_REQUESTED** [review]
Thank you very much!
LGTM except for my comment below 😉

File: modDesc.xml (hunk 1 above)
Comment (Alagoss) [reply modDesc.xml:104]: Added this bit =P

**Comment by bbenoist** [comment]
Merged with a single commit into master 🚀 
Thanks again for your work and efforts with GitHub!
Please let me know if they are remaining issues 😉

**Pull request closed without merging**